libpod/container_internal_linux.go

```diff
@@ -2159,11 +2159,24 @@ func (c *Container) getHosts() string {
 				}
 			}
 		} else if c.config.NetMode.IsSlirp4netns() {
-			gatewayIP, err := GetSlirp4netnsGateway(c.slirp4netnsSubnet)
-			if err != nil {
-				logrus.Warn("Failed to determine gatewayIP: ", err.Error())
-			} else {
-				hosts += fmt.Sprintf("%s host.containers.internal\n", gatewayIP.String())
+			// getLocalIP returns the non loopback local IP of the host
+			getLocalIP := func() string {
```

Comment:
Why is this a local function?

Reply:
Well it was local in Buildah, I will make it Public in Buildah and then we can use it in both places, but this will take a while.